lib/zendesk_apps_support/package.rb

```diff
@@ -365,5 +371,12 @@ def read_json(path, parser_opts = {})
       file = read_file(path)
       JSON.parse(read_file(path), parser_opts) unless file.nil?
     end
+
+    def text_file?(file)
+      fm = FileMagic.new(FileMagic::MAGIC_MIME)
+      fm.file(file.absolute_path) =~ /^text\/|json/
```

Comment:
I'm not really sure about this - should I instead filter out `charset=binary` in this string?

Reply:
Do we need to introduce a dependency on filemagic for this? What if instead we just matched on `.js, .html, .json` file extensions? Would we miss out any files that we care about?

Introducing a dependency on an external C library risks setup problems, especially since this gem is being included in a cross-platform CLI tool (ZAT)

Reply:
Yeah, I'm not sure. I originally wanted to scan all files that aren't binaries, but perhaps those file extensions are enough 🤷‍♀

FileMagic is probably a little heavy-handed in any case